packages/app/src/cli/services/context/identifiers-extensions.ts

```diff
@@ -26,7 +26,9 @@ export async function ensureExtensionsIds(
   const validIdentifiers = options.envIdentifiers.extensions ?? {}
   let localExtensions = options.app.allExtensions.filter((ext) => !ext.isFunctionExtension)
   if (options.deploymentMode === 'unified' || options.deploymentMode === 'unified-skip-release') {
-    const functionExtensions = options.app.allExtensions.filter((ext) => ext.isFunctionExtension)
+    const functionExtensions = options.app.allExtensions.filter(
+      (ext) => ext.isFunctionExtension && !ext.isConfigExtension,
```

Comment:
aren't this already mutually exclusive? or can a function be a config extension? 🤔

Reply:
Oops, let me fix it